tests/test_wave_01.py

```diff
@@ -118,13 +119,16 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
+    assert updated_data["watched"]
+    #added
+    assert updated_data["watched"][0]["title"] == MOVIE_TITLE_1
```

Comment:
Tests work better when they are comprehensive, so in this case I recommend checking all of the keys of the dictionary.  Checking the entire data structure ensures that the function being tested here didn't corrupt any of the data.
```suggestion
    assert updated_data["watched"][0]["title"] == MOVIE_TITLE_1
    assert updated_data["watched"][0]["genre"] == GENRE_1
    assert updated_data["watched"][0]["rating"] == RATING_1
```